check-node-version.js

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env node
+
+const minimumVersion = '12.0.0';
+const currentVersion = process.version.replace('v', '');
+
+const minimumVersionMajor = parseInt(minimumVersion.split('.')[0]);
+const currentVersionMajor = parseInt(currentVersion.split('.')[0]);
+
+const usesMinimumVersion = currentVersionMajor >= minimumVersionMajor;
+
+if (usesMinimumVersion) {
+  process.exit();
+}
+
+console.log(
```

Comment:
IMO this belongs on stderr:

```suggestion
console.error(
```